DuckDuckGo/AppDelegate.swift

```diff
@@ -326,6 +326,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         autoClear?.applicationDidEnterBackground()
         lastBackgroundDate = Date()
         AppDependencyProvider.shared.autofillLoginSession.endSession()
+        ContentBlocking.shared.contentBlockingManager.resetCompilationTime()
```

Comment:
This approach is fine for current compilation, but what if state of manager is `recompilingAndScheduled`, so subsequent one will be started shortly after?

Also alternative solution: maybe we should be starting background task during compilation so it completes just fine after entering background? Open question there is how would that affect potential reload triggered by it, while app is in the background.